### ISSUE TITLE
GI deprecation

### ISSUE THREAD

**Issue by grabear** [datasnakes/Datasnakes-Scripts#115]
NCBI has deprecated using GI numbers for anything in the future.  Here we must also do this.

In order to begin this process I have commented out code and created todos:

```python
# TODO: GI deprecation.
```
The following commits were made on individual files for deprecating the GI functionality:
* DatabaseManagement:  9713e87d1ebf29451ae91781a645029f3c91abe3
* get_gi_list.py: f9c595600aaae25ea71567203ce9a1384b535e91
* CompGenBlastn
    * 92db4ac7563a0b9b8549600355fce8887c305103
    * 8a20c32cbacfdfb0ddf10bf889fe51a7360cf4d4
* Blast README: 6a5ac79538546409f2d31d280469ed2bd1fc4dc3
* Blast\utils.py: 2f94c21e51b8fd4645cb404fceb0f154de332405

**Comment by grabear** [comment]
@sdhutchins GI list paths weren't integrated with the Manager module, and all of it's functionality.  So it's not necessary for anything to change at the moment.  Whenever the Window Masker stuff get's going, we can add an attribute into the management.py file and add the appropriate directories to cookiecutter templates.  It should go under <path>/users/<user>/databases/NCBI/<mimic-of-NCBI-ftp-path>/WindowMasker or something like that.

If you add some basic links here that would be awesome.

**Comment by grabear** [comment]
#113 to speed up BLAST we will deprecate GI numbers and use seqIDs or Windowmasker.

**Comment by sdhutchins** [comment]
I added this to the CompGenBLASTn class (now named OrthoBlastN) in the the `__init__`

```python
self.environment_vars = dict(os.environ)
if 'BLASTDB' not in self.environment_vars.keys():
    msg = "BLASTDB is not set in your path."
    raise EnvironmentError(msg)
elif 'WINDOW_MASKER_PATH' not in self.environment_vars.keys():
    msg = "WINDOW_MASKER_PATH is not set in your path."
    raise EnvironmentError(msg)       
```

**Comment by sdhutchins** [comment]
PR #116

**Comment by sdhutchins** [comment]
Addressed by #120
